pkg/server/image_pull.go

```diff
@@ -403,3 +410,15 @@ func newTransport() *http.Transport {
 		ExpectContinueTimeout: 5 * time.Second,
 	}
 }
+
+// addEncryptedImagesPullOpts adds the necessary pull options to a list of
+// pull options if enabled.
+func (c *criService) encryptedImagesPullOpts() []containerd.RemoteOpt {
+	if c.config.EncryptedImagesConfig.KeyModel == criconfig.EncryptionKeyModelNode {
```

Comment:
If `KeyModel == ""`, should it default to KeyModelNode?

Reply:
I was going to omit adding the additional flag if `[plugins.cri.image_encryption]` isn't going to be defined. I guess it's not that much of an overhead, so I am fine with defaulting to it.